recipes/emfile/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "emfile" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/emfile-{{ version }}.tar.gz
+  sha256: d7e9114aac5e2a29db9d77ab669659499c2f8c38ca13a5ff468c39ec89181a9b
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - numpy
+    - pip
+    - python >=3.7
+  run:
+    - numpy
+    - python >=3.7
+
+test:
+  imports:
+    - emfile
+  commands:
+    - pip list
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/brisvag/emfile
+  summary: Basic utility to read tomography data from files in `*.em` format.
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - brisvag
+    - conda-forge/napari
```

Comment:
This one does not depend on `napari`. Does the team need to be a maintainer here?

Reply:
(I am assuming this is a dependency we need for a plugin, right?)

Reply:
Correct